alerts/aladdin_connect_2.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Aladdin Connect integration will stop working January 23rd, 2024"
+created: 2024-01-19 00:00:01
+integrations:
+  - aladdin_connect
+---
+
+## Summary
+
+Genie is migrating to a new API on January 23rd, 2024, causing the Aladdin Connect integration in Home Assistant to stop working.
+
+Genie is planning to address this by contributing an official integration for Aladdin Connect to Home Assistant using the new API in the future.
```

Comment:
This will be raised as an issue in the end-user's repair center.

What does a user have to do to resolve this repair?

../Frenck